main.cpp

```diff
@@ -122,394 +172,476 @@ struct CarWash
+{
+    //int recordedNumber = number;
+    number = 0;
+    time++;
+}
+
+int CanPlant::OfficeWorker::reserveGood(int customerId, int amount)
+{
+    customerId++;
+    return amount++;
+}
+
+int CanPlant::produceCans(int fish, int tin)
+{
+    int amount = fish / tin;
+    return amount;
+}
+
+void CanPlant::loadBoxes(int boxes, int wareHouseNumber)
+{
+    boxes++;
+    wareHouseNumber++;
+}
+
+double CanPlant::reportScrapOut(double raw, double work)
+{
+    return raw * work *.01;
+}
+
+bool SipProvider::makeCall(int src, int dst)
+{
+    if (src == dst)
+    {
+        return false;
+    }
+    else
+    {
+        return true;
+    }
+}
+
+double SipProvider::chargeCustomer(double time, int customerId)
+{
+    if (customerId == 0)
+    {
+        return 0;
+    }
+    else
+    {
+        return time * price;
+    }
+}
+
+double SipProvider::traficCount(double time)
+{
+    return time * 0.1;
+}
+
+void Cat::eat(char foodType)
+{
+    if (foodType == 'F')
+    {
+        Cat::mew(3);
+    }
+}
+
+void Cat::sleep (float time)
+{
+    time--;
+}
+
+void Cat::mew (int count)
+{
+    count--;
+}
+
+void SpaceShip::CrewMember::examineAnimal(int date, float time, Cat cat)
+{
+    int testNum = 0;
+    if (cat.liveNumber <= 9)
+    {
+        recordTest(date, time, testNum++);
+    }
+    
+    date++;
+    time++;
+}
+
+bool SpaceShip::CrewMember::recordTest(int date, float time, int testNum)
+{
+    date++;
+    time++;
+    
+    if (testNum !=0)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+
+void SpaceShip::CrewMember::examineCrew (int date, float time, CrewMember id)
+{
+    int testNum = 0;
+    if (id.weight >= 70.0f)
+    {
+        recordTest(date, time, testNum++);
+        date++;
+        time++;
+    }
+}
+
+bool SpaceShip::dock()
+{
+    SpaceShip Rassvet;
+    if (Rassvet.orbitHeight != 40.0f)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+void SpaceShip::makeLoop(int planetNum )
+{
+    planetNum++;
+}
+
+bool SpaceShip::takeOf(float startTime)
+{
+    SpaceShip Rassvet;
+    startTime--;
+    if (Rassvet.orbitHeight != 0.0f)
+    {
+        return true;
+    } 
+    else
+    {
+        return false;
+    }
+}
+
+void DAC::readInput(int channelNum)
+{
+    for (int i=0;i<channelNum;i++)
+    {
+        checkError(i);
+    }
+}
+
+bool DAC::checkError(double sampleNum)
+{
+    double originalValue = 3.141592;
+    if (sampleNum > originalValue || sampleNum < originalValue)
+    {
+        return false;
+    }
+    else
+    {
+        return true;
+    }
+}
+
+void DAC::audioOut(int channelNum)
+{
+    HeadphoneAmp ampLeft, ampRight;
+    
+    //int bits = 24;
+    
+    for (int i=0;i<channelNum;i++)
+    {
+        ampLeft.doAmp(i);
+        ampRight.doAmp(i);
+    
+    }
+}
+
+bool PowerUnit::getElectricity(int outletStandart)
+{
+    if (outletStandart<3) return true;
+    return false;
+}
```

Comment:
add space around your operators
it is preferred if you put the `if()` body on its own line:
```
if( outletStandard < 3 )
    return true;
    
return false;
```

however, you can eliminate the `if()` entirely and write this instead:
```
return outletStandard < 3;
```
- [ ] revise